src/Extensions/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -37,6 +37,37 @@ public static class ActivityExtensions
 				HealthCheckMarkerValue,
 				StringComparison.OrdinalIgnoreCase);
 
+		/// <summary>
+		/// Returns true if activity is marked as Performance test
+		/// </summary>
+		/// <param name="activity"></param>
+		/// <returns>True if the activity is marked as a Performance test, false otherwise</returns>
+		/// <remarks>Currently added to the BaggageItems (via Kestrel) using header value Correlation-Context: "PerformanceTestMarker=0"</remarks>
+		public static bool IsPerformanceTest(this Activity activity) =>
+			!string.IsNullOrEmpty(activity.GetBaggageItem(PerformanceMarkerKey));
+
+		/// <summary>
+		/// Returns the value of the Performance key if it exists
+		/// </summary>
+		/// <param name="activity"></param>
+		/// <returns>The value passed for PerformanceTestMarker</returns>
+		/// <remarks>Setting the value in the BaggageItems, the interger value can be used to mock a delayed response from a partner call</remarks>
+		/// <remarks>Example: PerformanceTestMarker=500 mocking a 500 millisecond delay</remarks>
+		public static int? PerformanceValue(this Activity activity)
```

Comment:
Maybe this name would be more clear:
```suggestion
		public static int? PerformanceTestMakerValue(this Activity activity)
```